OpenRA.Mods.OpenE2140/Traits/Rendering/WithMovingSpriteTurret.cs

```diff
@@ -57,9 +62,9 @@ protected override WVec TurretOffset(Actor self)
 		if (notWalking)
 			return offset;
 
-		var interpolation = int2.Lerp(this.info.OffsetLow, this.info.OffsetHigh, this.current, this.info.Duration);
+		var interpolation = int2.Lerp(this.info.OffsetLow, this.info.OffsetHigh, this.current, this.duration);
```

Comment:
```suggestion
var duration = this.mobile.CurrentMovementTypes is MovementType.Turn ? this.info.TurnDuration : this.info.Duration;
var interpolation = int2.Lerp(this.info.OffsetLow, this.info.OffsetHigh, this.current, duration);
```

Suggestion for calculating duration (see comment for `duration` field above).